manifests/dotnet.yml

```
@@ -24,6 +24,8 @@ tests/:
           TestInsecureCookie: missing_feature
         test_ldap_injection.py:
           TestLDAPInjection: missing_feature
+        test_multpart.py:
```

Typo in all manifests: 

```suggestion
        test_multipart.py:
```

---

The typo is also on other manifest files.